cmake/qemu/CMakeLists.txt

```diff
@@ -30,6 +30,16 @@ if(EMU_PLATFORM)
     endif()
   endif()
 
+  # If we are running a networking application in QEMU, then set proper
+  # QEMU variables. This also allows two QEMUs to be hooked together and
+  # pass data between them. The QEMU flags are not set for standalone
+  # tests defined by CONFIG_NET_TEST.
+  if(CONFIG_NETWORKING)
+    if(!CONFIG_NET_TEST)
```

Comment:
Huh, does that work? Thought you had to use "NOT".

https://cmake.org/cmake/help/v3.0/command/if.html

Reply:
Hmm, I did not knew that, worked just fine when I used it :)

Reply:
Actually, it does not seem to work properly. I will send fixed version.